djangoform/views.py

```diff
@@ -38,9 +38,17 @@ def create_recipe(request):
 def recipe_info(request, id):
     html = 'recipe_info.html'
     items = Recipes.objects.all().filter(id=id)
+    author = Recipes.objects.all().filter(id=id).values_list('id', flat=True).first()
```

Comment:
the `author` variable is currently returning the id of the recipe you are targeting. This is not the id we want to check `request.user.id` against on line 51.
```suggestion
    author = Recipes.objects.filter(id=id).author.user.id
```